tests/components/alarm_control_panel/common.py

```diff
@@ -61,6 +62,19 @@ async def async_alarm_arm_night(hass, code=None, entity_id=ENTITY_MATCH_ALL):
     await hass.services.async_call(DOMAIN, SERVICE_ALARM_ARM_NIGHT, data, blocking=True)
 
 
+async def async_alarm_arm_vacation(hass, code=None, entity_id=ENTITY_MATCH_ALL):
```

Comment:
Please don't add this helper. We're trying to remove these helpers. Just call the service directly.

Reply:
If i remove this it breaks the tests. If this needs to be removed please provide replacement code as i don't have the knowlegde how to solve

Reply:
fixed with correction support integers

Reply:
Where is this used?

Reply:
helper removed tnx to @KJonline